packages/quick_actions/quick_actions/example/ios/RunnerUITests/RunnerUITests.m

```diff
@@ -11,18 +11,24 @@ @interface RunnerUITests : XCTestCase
 
 @end
 
-@implementation RunnerUITests
+@implementation RunnerUITests {
+  XCUIApplication *_exampleApp;
+}
 
 - (void)setUp {
   [super setUp];
   self.continueAfterFailure = NO;
+  _exampleApp =
+      [[XCUIApplication alloc] initWithBundleIdentifier:@"io.flutter.plugins.quickActionsExample"];
```

Comment:
Why do we need to specify the app by bundle ID?

Reply:
Not necessarily, however since we also reference the "Springboard" App in the tests I found it easier (more explicit) to reference the example App through the bundle identifier as wel.

Otherwise it would read:

```
_exampleApp = [[XCUIApplication alloc] init];
```

And we need to assume this is the example app.